src/components/InputForm.js

```diff
@@ -0,0 +1,246 @@
+import React, { useState } from 'react'
+import { yupResolver } from '@hookform/resolvers'
+import * as yup from 'yup'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import { FiArrowLeftCircle } from 'react-icons/fi'
+import { action } from '@storybook/addon-actions'
+
+const projectSchema = yup.object().shape({
+  name: yup.string().min(5).max(30).required(),
+  pointPurchase: yup.string().min(10).max(30).required(),
+  description: yup.string().min(10).max(300).required(),
+})
+
+export default function InputForm({ setTea }) {
+  const [newTea, setNewTea] = useState({
+    name: '',
+    pointPurchase: '',
+    description: '',
+    resolver: yupResolver(projectSchema),
+  })
+
+  return (
+    <main>
+      <StyledForm onSubmit={handleSubmit}>
+        <StyledLink to="/tealistpage">
```

Comment:
Schau mal ob du den Link Button in deine übergeordnete Page bekommst, wo du die Form hier aufrufst, damit du in dieser Komponente wirklich nur die Sachen drin hast, die zur Form gehören.